diff --git a/packages/dpki-lite/lib/util.js b/packages/dpki-lite/lib/util.js
index 16c7e31..2d6cb6e 100644
--- a/packages/dpki-lite/lib/util.js
+++ b/packages/dpki-lite/lib/util.js
@@ -93,7 +93,7 @@ function pwHash(pass, salt) {
     _sodium.ready.then((_) => {
       const opt = {
         opslimit: _sodium.crypto_pwhash_OPSLIMIT_SENSITIVE,
-        memlimit: _sodium.crypto_pwhash_MEMLIMIT_SENSITIVE,
+        memlimit: _sodium.crypto_pwhash_MEMLIMIT_INTERACTIVE,
         algorithm: _sodium.crypto_pwhash_ALG_ARGON2ID13,
         keyLength: 32
       }
